scripts/upload-housing.js

```diff
@@ -117,17 +122,18 @@ const main = async () => {
   const db = getFirestore(app);
 
   // Check if the collection exists
+  // TODO how do i make this check for housing instead of spotlight
```

Comment:
You could look into getting all the documents in a collection? And then check if the doc id matches what you want?

https://firebase.google.com/docs/firestore/query-data/get-data#get_all_documents_in_a_collection